docs/QUICKSTART.md

```diff
@@ -0,0 +1,403 @@
+# Quick-Start Guide for uVisor on mbed OS
+
+This guide will help you get started with uVisor on mbed OS.
+
+The uVisor creates sandboxed environments and resources protection for application built for ARM Cortex-M3 and Cortex-M4 devices. Here we will show you how to enable the uVisor and configure a secure box to get hold of some exclusive resources (memory, peripherals, interrupts). For more information on the uVisor design philosophy, please check out our the uVisor [introductory document](../README.md).
```

Comment:
I like "uVisor" much more than "the uVisor", but am not sure if there was a "marketing" decision to use the article. @meriac?